bootstrap-salt.sh

```diff
@@ -1884,7 +1884,7 @@ install_ubuntu_stable_deps() {
         __apt_get_install_noinput wget
 
         # shellcheck disable=SC2086
-        wget $_WGET_ARGS -q http://repo.saltstack.com/apt/ubuntu/ubuntu$DISTRO_MAJOR_VERSION/$STABLE_REV/SALTSTACK-GPG-KEY.pub -O - | apt-key add - || return 1
+        wget $_WGET_ARGS -q https://repo.saltstack.com/apt/ubuntu/ubuntu$DISTRO_MAJOR_VERSION/$STABLE_REV/SALTSTACK-GPG-KEY.pub -O - | apt-key add - || return 1
```

Comment:
@The-Loeki, this should be `https://repo.saltstack.com/apt/ubuntu/$DISTRO_VERSION/$CPU_ARCH/$STABLE_REV/SALTSTACK-GPG-KEY.pub`.